resolver/uuidResolver.go

```diff
@@ -0,0 +1,77 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/uuid-utils-go"
+	"time"
+)
+
+const dateTimeFormat = "2006-01-02T15:04:05.000Z0700"
+
+type UuidsAndDateResolver interface {
+	Resolve(reqData []byte, respData []byte) (*UuidsAndDate, error)
+}
+
+type UuidsAndDate struct {
+	UuidArr      []string
```

Comment:
The encouraged naming here would be to use upper case for abbreviations (UUID). Check for more like this through the app.